codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/customization/presignable/PresignableUrlIntegration.kt

```diff
@@ -204,6 +210,10 @@ class PresignableUrlIntegration(private val presignedOperations: Map<String, Set
                 operationMiddlewareCopy.removeMiddleware(op, "OperationInputBodyMiddleware")
                 operationMiddlewareCopy.appendMiddleware(op, PutObjectPresignedURLMiddlewareRenderable())
             }
+            "com.amazonaws.s3#UploadPart" -> {
+                operationMiddlewareCopy.removeMiddleware(op, "OperationInputBodyMiddleware")
+                operationMiddlewareCopy.appendMiddleware(op, UploadPartPresignedURLMiddlewareRenderable())
+            }
```

Comment:
This is where you could just add back the `QueryItemMiddleware` which has the name `OperationInputQueryItemMiddleware`; it's located [here](https://github.com/smithy-lang/smithy-swift/blob/10de568b01aa71134e22e6253525afb9d0f4f8cf/smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/middlewares/OperationInputQueryItemMiddleware.kt#L12).